src/languages/es.js

```diff
@@ -123,6 +123,7 @@ export default {
         areYouSure: '¿Estás seguro?',
         zipCodeExample: 'p. ej. 12345, 12345-1234, 12345 1234',
         websiteExample: 'p. ej. https://www.expensify.com',
+        format: 'Formato:',
```

Comment:
as i mentioned in the `en.js` file, i think we need to handle the format param here?